### ISSUE TITLE
Doesn't work with the last ruby-runtime

### ISSUE THREAD

**Issue by pcollaog** [elvanja/jenkins-gitlab-hook-plugin#13]
Jenkins use the new ruby-runtime 0.11 and the jenkins-gitlab-hook-plugins needs the ruby-runtime 0.10. 

Finally the plugin doesn't install with error.

Version of plugin: 0.2.6

```
INFO: Attempting to dynamic load /opt/jenkins/plugins/gitlab-hook.jpi
jun 30, 2013 3:24:42 PM hudson.model.UpdateCenter$DownloadJob run
SEVERE: Failed to install Gitlab Hook Plugin
hudson.util.IOException2: Failed to dynamically deploy this plugin
        at hudson.model.UpdateCenter$InstallationJob._run(UpdateCenter.java:1314)
        at hudson.model.UpdateCenter$DownloadJob.run(UpdateCenter.java:1111)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
        at java.util.concurrent.FutureTask.run(FutureTask.java:166)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:722)
Caused by: hudson.util.IOException2: Failed to install gitlab-hook plugin
        at hudson.PluginManager.dynamicLoad(PluginManager.java:417)
        at hudson.model.UpdateCenter$InstallationJob._run(UpdateCenter.java:1310)
        ... 7 more
Caused by: hudson.util.IOException2: Failed to initialize
        at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:379)
        at hudson.PluginManager.dynamicLoad(PluginManager.java:408)
        ... 8 more
Caused by: java.lang.Exception: unable to locate gem bundle for gitlab-hook at /opt/jenkins/plugins/gitlab-hook/WEB-INF/classes/vendor/gems
        at ruby.RubyPlugin.initRubyLoadPaths(RubyPlugin.java:202)
        at ruby.RubyPlugin.start(RubyPlugin.java:158)
        at hudson.ClassicPluginStrategy.startPlugin(ClassicPluginStrategy.java:387)
        at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:376)
        ... 9 more
```

**Comment by qadeerqureshi** [comment]
Hi, i am using Jenkins 2.58 version and i am having error. Please help thanks! 


```
Failed Loading plugin ruby-runtime v0.12 (ruby-runtime)
org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- haml
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1027)
	at RUBY.require(jar:file:/C:/Program%20Files%20(x86)/Jenkins/plugins/ruby-runtime/WEB-INF/lib/jruby-complete-1.7.3.jar!/META-INF/jruby.home/lib/ruby/shared/rubygems/custom_require.rb:36)
	at RUBY.(root)(file:/C:/Program Files (x86)/Jenkins/plugins/ruby-runtime/WEB-INF/lib/stapler-jruby-1.209.jar!/org/kohsuke/stapler/jelly/jruby/haml/JRubyJellyHamlScript.rb:1)
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1027)
	at RUBY.(root)(jar:file:/C:/Program%20Files%20(x86)/Jenkins/plugins/ruby-runtime/WEB-INF/lib/jruby-complete-1.7.3.jar!/META-INF/jruby.home/lib/ruby/shared/rubygems/custom_require.rb:1)
	at RUBY.require(jar:file:/C:/Program%20Files%20(x86)/Jenkins/plugins/ruby-runtime/WEB-INF/lib/jruby-complete-1.7.3.jar!/META-INF/jruby.home/lib/ruby/shared/rubygems/custom_require.rb:36)
Caused: org.jruby.embed.EvalFailedException: (LoadError) no such file to load -- haml
	at org.jruby.embed.internal.EmbedEvalUnitImpl.run(EmbedEvalUnitImpl.java:133)
	at org.jruby.embed.ScriptingContainer.runUnit(ScriptingContainer.java:1264)
	at org.jruby.embed.ScriptingContainer.runScriptlet(ScriptingContainer.java:1257)
	at org.kohsuke.stapler.jelly.jruby.haml.HamlLanguage.createContainer(HamlLanguage.java:28)
	at org.kohsuke.stapler.jelly.jruby.JRubyFacet.<init>(JRubyFacet.java:71)
	at ruby.RubyRuntimePlugin.registerJRubyFacet(RubyRuntimePlugin.java:39)
	at ruby.RubyRuntimePlugin.start(RubyRuntimePlugin.java:30)
	at hudson.ClassicPluginStrategy.startPlugin(ClassicPluginStrategy.java:532)
	at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:521)
Caused: java.io.IOException: Failed to initialize
	at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:524)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:517)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)
	at jenkins.model.Jenkins$7.runTask(Jenkins.java:1088)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
```

**Comment by wcdeich4** [comment]
But I can't get ruby-runtime v0.12 to install.  
SEVERE: Failed Loading plugin ruby-runtime v0.12 (ruby-runtime)
java.io.IOException: Failed to initialize
	at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:528)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:517)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)
	at jenkins.model.Jenkins$7.runTask(Jenkins.java:1073)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.jruby.embed.EvalFailedException: (LoadError) no such file to load -- haml
	at org.jruby.embed.internal.EmbedEvalUnitImpl.run(EmbedEvalUnitImpl.java:133)
	at org.jruby.embed.ScriptingContainer.runUnit(ScriptingContainer.java:1264)
	at org.jruby.embed.ScriptingContainer.runScriptlet(ScriptingContainer.java:1257)
	at org.kohsuke.stapler.jelly.jruby.haml.HamlLanguage.createContainer(HamlLanguage.java:28)
	at org.kohsuke.stapler.jelly.jruby.JRubyFacet.<init>(JRubyFacet.java:71)
	at ruby.RubyRuntimePlugin.registerJRubyFacet(RubyRuntimePlugin.java:39)
	at ruby.RubyRuntimePlugin.start(RubyRuntimePlugin.java:30)
	at hudson.ClassicPluginStrategy.startPlugin(ClassicPluginStrategy.java:536)
	at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:525)
	... 9 more
Caused by: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- haml
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1027)
	at RUBY.require(jar:file:/C:/Program%20Files%20(x86)/Jenkins/plugins/ruby-runtime/WEB-INF/lib/jruby-complete-1.7.3.jar!/META-INF/jruby.home/lib/ruby/shared/rubygems/custom_require.rb:36)
	at RUBY.(root)(file:/C:/Program Files (x86)/Jenkins/plugins/ruby-runtime/WEB-INF/lib/stapler-jruby-1.209.jar!/org/kohsuke/stapler/jelly/jruby/haml/JRubyJellyHamlScript.rb:1)
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1027)
	at RUBY.(root)(jar:file:/C:/Program%20Files%20(x86)/Jenkins/plugins/ruby-runtime/WEB-INF/lib/jruby-complete-1.7.3.jar!/META-INF/jruby.home/lib/ruby/shared/rubygems/custom_require.rb:1)
	at RUBY.require(jar:file:/C:/Program%20Files%20(x86)/Jenkins/plugins/ruby-runtime/WEB-INF/lib/jruby-complete-1.7.3.jar!/META-INF/jruby.home/lib/ruby/shared/rubygems/custom_require.rb:36)

Sep 13, 2017 9:06:25 PM jenkins.InitReactorRunner$1 onTaskFailed
SEVERE: Failed Loading plugin Gitlab Hook Plugin v1.4.2 (gitlab-hook)
java.io.IOException: Gitlab Hook Plugin v1.4.2 failed to load.
 - ruby-runtime v0.12 is missing. To fix, install v0.12 or later.
	at hudson.PluginWrapper.resolvePluginDependencies(PluginWrapper.java:626)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:516)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)
	at jenkins.model.Jenkins$7.runTask(Jenkins.java:1073)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
